docs/graphene-workload-tutorial/README.md

```diff
@@ -0,0 +1,62 @@
+<!--
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+-->
+
+# Writing a Python workload to run with Graphene SGX
+
+## Changes to customize the template
+
+For this tutorial let us consider a check for palindrome as the business logic for our workload.
+Bring up the docker container from current directory with the name of your workload as an environment variable. For example -
+```
+WORKLOAD_NAME=palindrome docker-compose up
+```
+This will prepare your the setup for building your workload. You just need to add the business logic.
+
+### Source code changes in template.py
+
+Uncomment all code blocks in the function `_palindrome()`. This function holds the core business logic.
+
+This completes the business logic implementation.
+
+### Setup Changes
+
+Update the `workload/tests/test_work_orders.json` file if you want to add/remove/update test cases.
+
+### Avalon specific changes
+
+Update image name in the docker compose files `docker/compose/avalon-fib-graphene.yaml` and `docker/compose/avalon-fib-gsgx.yaml`. You could rename to these files to `docker/compose/avalon-pal-graphene.yaml` and `docker/compose/avalon-pal-gsgx.yaml` respectively as well.
```

Comment:
Let's create `avalon-workload-graphene.yaml.template` and `avalon-workload-gsgx.yaml.template` and then generate avalon workload files at location `${TCF_HOME}/docker/compose`